### ISSUE TITLE
Unable to get the code working

### ISSUE THREAD

**Issue by thak123** [kawu/nerf#49]
How to get this library to work in 2019. None of the dependencies are getting compiled.

Can anyone help

**Comment by kawu** [comment]
Performing the steps described in the [installation section](https://github.com/kawu/nerf#installation) of the README (in particular, `stack install`) works for me.  All the dependencies get compiled.

Could you please give me some more details about your problem?  Maybe you are trying to use the library as a part of a larger project, and the tool's dependencies (admittedly outdated) cause the issue?  If so, I can try to update them.

**Comment by thak123** [comment]
Yes Sir... I am trying to use this a part of my NER pipeline.  Can you please update the dependencies to the latest so that stack install is smooth enough.

**Comment by kawu** [comment]
Hi, sorry it took so long, but I had to update old versions of several libraries that Nerf relies on...  In any case, Nerf should now work with the newest version of Haskell stack (I checked with [LTS 14.16](https://www.stackage.org/lts-14.16)).  It's also uploaded to Hackage.
